bench/benchmark.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var benchmark = require('benchmark');
+var Table = require('cli-table');
+var colors = require('colors/safe');
```

Comment:
I'd like to avoid this dependency. Defining `colors.red` and `colors.green` ourselves is trivial:

```javascript
function red(s) {
  return '\u001B[31m' + s + '\u001B[0m';
}

function green(s) {
  return '\u001B[32m' + s + '\u001B[0m';
}
```

The useful thing the dependency does for us is determine whether it's appropriate to use colour, in [__supports-colors.js__][1]. The logic could be simplified and abbreviated to the following:

```javascript
var isColorTerm =
  process.stdout.isTTY &&
  /^screen|^xterm|^vt100|color|ansi|cygwin|linux/i.test(process.env.TERM);
```

Perhaps `isColorTerm` should be determined in __bench/index.js__ and provided to `runBenchmarks` as an argument. We could then change this code:

```diff
-      var colorize = change > +0.1 ? colors.green :
-                     change < -0.1 ? colors.red :
+      var colorize = isColorTerm && change > +0.1 ? green :
+                     isColorTerm && change < -0.1 ? red :
                      identity;
```


[1]: https://github.com/Marak/colors.js/blob/v1.1.2/lib/system/supports-colors.js